### PR TITLE
usage documentation update

### PR DESCRIPTION
**Pull request by sanyam803**
Update documentation to reflect private launch scope & postures submitted currently.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,149 +1,161 @@
 # analyze-code-security-scc
 
+
 ## Description
 
-This Github action identifies insecure configurations in Infrastructure as Code (IaC) files for Google Cloud resources. 
-This action requires Terraform plan files in JSON format for scanning.
+
+This Github action identifies insecure configurations in Infrastructure as Code (IaC) files for Google Cloud resources. This action requires Terraform plan files in JSON format for scanning.
```
Comment (sethvargo) [inline README.md:7]: Capital "H" in GitHub (throughout)

File: README.md [hunk 2]
```diff
@@ -1,149 +1,161 @@
...
 - Display issues with their severity as a SARIF Report in the GitHub Workspace after a scan completes.
 - Define severity-based failure criteria for passing or failing the build.
 
-> [!NOTE] 
-> This is a Security Command Center Premium tier offering for subscription customers only. You must activate the Security Command Center Premium tier in the Google Cloud organization to use this feature.
 
-**This is not an officially supported Google product, and it is not covered by a
-Google Cloud support contract. To report bugs or request features in a Google
-Cloud product, please contact [Google Cloud
-support](https://cloud.google.com/support).**
```
Comment (verbanicm) [inline README.md:21]: @sethvargo do we want to keep this messaging in here? I believe yes...

File: README.md [hunk 3]
```diff
@@ -1,149 +1,161 @@
...
+Note: This is a Security Command Center Premium tier offering for subscription customers only. You must activate the Security Command Center Premium tier in the Google Cloud organization to use this feature.
```
Comment (verbanicm) [inline README.md:19]: Id prefer to keep the GitHub [markdown alert](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts) syntax. This helps call this out to users.

```markdown
> [!IMPORTANT]
> This is a Security Command Center Premium tier offering for subscription customers only. You must activate the Security Command Center Premium tier in the Google Cloud organization to use this feature.
> 
> This offering is covered by the Pre-GA Offerings Terms of the Google Cloud Terms of Service.

> [!CAUTION]
> Don’t enter any sensitive information such as passwords and other personal identifiable information in the Terraform plan files.
```

Example
> [!IMPORTANT]
> This is a Security Command Center Premium tier offering for subscription customers only. You must activate the Security Command Center Premium tier in the Google Cloud organization to use this feature.
> 
> This offering is covered by the Pre-GA Offerings Terms of the Google Cloud Terms of Service.

> [!CAUTION]
> Don’t enter any sensitive information such as passwords and other personal identifiable information in the Terraform plan files.

File: README.md [hunk 4]
```diff
@@ -1,149 +1,161 @@
...
+       ignore_violations: false
+       failure_criteria: 'High:1,Medium:1,Low:1,Operator:or'
+       fail_silently: false
+
+
+   - if: ${{steps.analyze-code-security-scc.outputs.iac_scan_result_sarif_path != ''}}
+       uses: actions/upload-artifact@v4
+       with:
+         name: sarif
+         path: ${{ steps.analyze-code-security-scc.outputs.iac_scan_result_sarif_path }}
```
Comment (verbanicm) [inline README.md:77]: Make sure all non-boolean yaml attributes are wrapped in single quotes (throughout)

```suggestion
         path: '${{ steps.analyze-code-security-scc.outputs.iac_scan_result_sarif_path }}'
```

File: README.md [hunk 5]
```diff
@@ -1,149 +1,161 @@
...
+       scan_file_ref: './tf_plan.json'
+       iac_type: terraform
+       iac_version: '1.0.0'
+       scan_timeout: '1m'
+       ignore_violations: false
+       failure_criteria: 'High:1,Medium:1,Low:1,Operator:or'
+       fail_silently: false
+
+
+   - if: ${{steps.analyze-code-security-scc.outputs.iac_scan_result_sarif_path != ''}}
```
Comment (verbanicm) [inline README.md:73]: You can use multi-line strings to make this cleaner:

```suggestion
   - if: |-
     steps.analyze-code-security-scc.outputs.iac_scan_result_sarif_path != ''
```

File: README.md [hunk 6]
```diff
@@ -1,149 +1,161 @@
...
+
+   - id: 'analyze-code-security-scc'
+     uses: 'google-github-actions/analyze-code-security-scc@v1'
 ```
 
+
+## Supported asset types and policies
+
+
+For information about supported asset types and policies, see IaC Validation - Supported assets and policies.
```
Comment (verbanicm) [inline README.md:157]: Should there a link to somewhere here?

File: README.md [hunk 7]
```diff
@@ -1,149 +1,161 @@
...
+   steps:
+   - id: 'auth'
+     uses: 'google-github-actions/auth@v2'
+     with:
+       workload_identity_provider: 'projects/123456789/locations/global/workloadIdentityPools/my-pool/providers/my-provider'
+       service_account: 'my-service-account@my-project.iam.gserviceaccount.com'
+
+
+   - id: 'analyze-code-security-scc'
+     uses: 'google-github-actions/analyze-code-security-scc@v1'
```
Comment (verbanicm) [inline README.md:150]: I believe this should be `v0`

```suggestion
     uses: 'google-github-actions/analyze-code-security-scc@v0'
```

File: README.md [hunk 8]
```diff
@@ -1,149 +1,161 @@
...
+   - uses: 'actions/checkout@v4' 
+   - id: 'auth'
+     uses: 'google-github-actions/auth@v2'
+     with:
+       workload_identity_provider: 'projects/123456789/locations/global/workloadIdentityPools/my-pool/providers/my-provider'
+       service_account: 'my-service-account@my-project.iam.gserviceaccount.com'
+
+
+   - id: 'analyze-code-security-scc'
+     uses: 'google-github-actions/analyze-code-security-scc@v1'
```
Comment (verbanicm) [inline README.md:61]: I believe this should be `v0`

```suggestion
     uses: 'google-github-actions/analyze-code-security-scc@v0'
```

File: README.md (hunk 2 above)
Comment (sethvargo) [reply README.md:21]: Yes, we need to include this.

File: README.md (hunk 8 above)
Comment (sanyam803) [reply README.md:61]: Done.

File: README.md (hunk 6 above)
Comment (sanyam803) [reply README.md:157]: Thanks missed the link. Added the link.

File: README.md (hunk 5 above)
Comment (sanyam803) [reply README.md:73]: Done.

File: README.md (hunk 4 above)
Comment (sanyam803) [reply README.md:77]: Done.

File: README.md (hunk 3 above)
Comment (sanyam803) [reply README.md:19]: Done.

File: README.md (hunk 1 above)
Comment (sanyam803) [reply README.md:7]: Done.

File: README.md (hunk 2 above)
Comment (verbanicm) [reply README.md:21]: @sanyam803 Can you add this messaging back in to the readme?

File: README.md [hunk 9]
```diff
@@ -1,149 +1,165 @@
...
+       failure_criteria: 'High:1,Medium:1,Low:1,Operator:or'
+       fail_silently: false
+
+
+   - if: |-
+       steps.analyze-code-security-scc.outputs.iac_scan_result_sarif_path != ''
+       uses: 'actions/upload-artifact@v4'
+       with:
+         name: 'sarif'
+         path: '${{ steps.analyze-code-security-scc.outputs.iac_scan_result_sarif_path }}'
```
Comment (verbanicm) [inline README.md:81]: I think the indentation is off here? Starting at `uses`

Comment (verbanicm) [inline README.md:81]:
```suggestion
jobs:
  job_id:
    permissions:
      contents: 'read'
      id-token: 'write'

    steps:
      - uses: 'actions/checkout@v4'
      - id: 'auth'
        uses: 'google-github-actions/auth@v2'
        with:
          workload_identity_provider: 'projects/123456789/locations/global/workloadIdentityPools/my-pool/providers/my-provider'
          service_account: 'my-service-account@my-project.iam.gserviceaccount.com'

      - id: 'analyze-code-security-scc'
        uses: 'google-github-actions/analyze-code-security-scc@v0'
        with:
          organization_id: '123456789'
          scan_file_ref: './tf_plan.json'
          iac_type: 'terraform'
          iac_version: '1.0.0'
          scan_timeout: '1m'
          ignore_violations: false
          failure_criteria: 'High:1,Medium:1,Low:1,Operator:or'
          fail_silently: false

      - if: |-
          steps.analyze-code-security-scc.outputs.iac_scan_result_sarif_path != ''
        uses: 'actions/upload-artifact@v4'
        with:
          name: 'sarif'
          path: '${{ steps.analyze-code-security-scc.outputs.iac_scan_result_sarif_path }}'

```

**Review by verbanicm — APPROVED** [review]

**Pull request merged**